dt-mapping/mapping-queries.php

```diff
@@ -1301,34 +1301,72 @@ public static function post_type_geojson( $post_type, $args = [], $offset = 0, $
         global $wpdb;
 
         //phpcs:disable
+        // Determine if post id filtering should take place.
+        $post_id_filter_sql = '';
+        $shared_user_join_sql = '';
+        $shared_user_condition_sql = '';
+        if ( isset( $args['list_all'], $args['list_all_by_user_id'] ) && !$args['list_all'] ) {
+            $post_id_filter_sql = $wpdb->prepare( "
+            SELECT * FROM
+                      (
+                        SELECT api_p.ID
+                        FROM $wpdb->posts api_p
+                        LEFT JOIN $wpdb->postmeta as field_type ON ( field_type.post_id = api_p.ID AND field_type.meta_key = 'type' )
+                        WHERE (field_type.meta_value = 'access' OR field_type.meta_value = 'user' OR field_type.meta_value = 'access_placeholder')
+                          AND (api_p.post_status = 'publish')
+                          AND api_p.post_type = %s
+                          GROUP BY api_p.ID
+                          LIMIT 0, 50000
+                      ) AS assigned_post_ids
```

Comment:
I'm confused at this whole section. It says assigned_post_ids, but no sql to to with assignment?
Could be that just the share filter is needed here.